eks-worker-al2.json

```diff
@@ -202,13 +204,19 @@
       "type": "shell",
       "remote_folder": "{{ user `remote_folder`}}",
       "script": "{{template_dir}}/scripts/generate-version-info.sh",
-      "execute_command": "chmod +x {{ .Path }}; {{ .Path }} /tmp/version-info.json"
+      "execute_command": "chmod +x {{ .Path }}; {{ .Path }} {{user `working_dir`}}/version-info.json"
     },
     {
       "type": "file",
       "direction": "download",
-      "source": "/tmp/version-info.json",
+      "source": "{{user `working_dir`}}/version-info.json",
       "destination": "{{ user `ami_name` }}-version-info.json"
+    },
+    {
+      "type": "shell",
+      "inline": [
+        "rm -rf {{user `working_dir`}}"
```

Comment:
Is it possible bad things will happen here? I think not since we're creating `{{user `remote_folder`}}/worker`? I don't how we could accidentally delete anything too problematic.